### PR TITLE
Ports Changelog generation script for Github Actions

### PR DESCRIPTION
**Pull request by quardbreak**
~~Initially idea credits to @AlexMorgan3817 (?), ported from Infinity Station and TGMC.~~
Ported from CEV Eris.

## Changelog:
:cl:
add: Added actions for changelog generation from Eris.
/:cl:

### CODE REVIEW

**Comment by AlexMorgan3817** [comment]
:eyes:

File: .github/workflows/changelog_generation.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+on:
+  push:
+    branches:
+    - dev
+
+jobs:
+  Compile_Changelog:
+    runs-on: ubuntu-latest
+    steps:
```
Comment (AlexMorgan3817) [inline .github/workflows/changelog_generation.yml]: Where
```suggestion
    runs-on: ubuntu-latest
    if: github.repository == 'NebulaSS13/Nebula'
    steps:
```

Comment (quardbreak) [reply .github/workflows/changelog_generation.yml]: checked the test script, I think this not needed to work

File: .github/workflows/changelog_generation.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+name: Compile changelogs
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
```
Comment (AlexMorgan3817) [inline .github/workflows/changelog_generation.yml]:
```suggestion
on:
  push:
    branches:
    - dev
```
Or say why you need to schedule it by cron

Comment (AlexMorgan3817) [reply .github/workflows/changelog_generation.yml]: Ладно, зря быканул

Comment (eckff) [reply .github/workflows/changelog_generation.yml:5]: > master
ok

**Review by PsiOmegaDelta — APPROVED** [review]

**Pull request merged**